serializers/protobuf/src/main/java/io/pravega/schemaregistry/serializer/protobuf/schemas/ProtobufSchema.java

```diff
@@ -153,6 +153,7 @@ public static <T extends GeneratedMessageV3> ProtobufSchema<GeneratedMessageV3>
      *
      * @param schemaInfo              Schema Info
      * @return {@link ProtobufSchema} with generic type {@link DynamicMessage} that captures protobuf schema.
+     * @throws IllegalArgumentException
```

Comment:
method signature needs to be changed too?

Reply:
As this method is throwing this exception, claimed by checkstylel.

Reply:
if this method is throwing the exception, this should be added to the method signature as well. And how are we handling that exception?

Reply:
Actually this method was already throwing  IllegalArgumentException as this is a unchecked so it was not added in method signature.

Check style was claiming if we are not showing this in Doc.

Reply:
This is a unchecked exception so doesn't require a throws clause, so why are we mentioning it in docs? Why check style is mandating to place this exception in docs ?

Reply:
Checkstyle is mandating it because `InvalidProtocolBufferException` this exception is raising its throwing null pointer exception.
other I there is a way we can hide this from documentation by using hyphen after the exception, what it will do while generating doc it will remove this exception from the doc.

but i am not getting a way in which i just remove this throws from the method doc.

either checked or unchecked its being thrown by the method.